### PR TITLE
Remove references to Rails 4 from assets guide [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

File: guides/source/asset_pipeline.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ What is the Asset Pipeline?
 The asset pipeline provides a framework to concatenate and minify or compress
 JavaScript and CSS assets. It also adds the ability to write these assets in
 other languages and pre-processors such as CoffeeScript, Sass and ERB.
-It allows assets in your application to be automatically combined with assets 
+It allows assets in your application to be automatically combined with assets
 from other gems. For example, jquery-rails includes a copy of jquery.js
 and enables AJAX features in Rails.
 
-The asset pipeline is technically no longer a core feature of Rails 4, it has
+The asset pipeline is technically no longer a core feature from Rails 4 onwards, it has
```
Comment (maclover7) [inline guides/source/asset_pipeline.md]: while the PR is open, can you change the comma to a "--"?

**Comment by prathamesh-sonpatki** [comment]
@maclover7 done :)

File: guides/source/asset_pipeline.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ What is the Asset Pipeline?
 The asset pipeline provides a framework to concatenate and minify or compress
 JavaScript and CSS assets. It also adds the ability to write these assets in
 other languages and pre-processors such as CoffeeScript, Sass and ERB.
-It allows assets in your application to be automatically combined with assets 
+It allows assets in your application to be automatically combined with assets
```
Comment (vipulnsward) [inline guides/source/asset_pipeline.md]: no changes here.

Comment (prathamesh-sonpatki) [inline guides/source/asset_pipeline.md]: It's whitespace change and do we allow such changes when we are updating a file without having too much git churn.

**Comment by prathamesh-sonpatki** [comment]
Merging this, thanks for the review everyone.

**Pull request merged**